src/main/java/com/google/sps/servlets/ChatroomServlet.java

```diff
@@ -50,7 +50,7 @@
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-        String userID = request.getParameter("userId");
+        String userID = request.getHeader("userId");
```

Comment:
It this something left out from previous changes? As it's not  a test file.

Reply:
Yes. It slipped past me